pyvista/plotting/plotting.py

```diff
@@ -260,7 +260,7 @@ def __init__(self, shape=(1, 1), border=None, border_color='k',
         self.reset_key_events()
 
     #### Manage the active Renderer ####
-    
```

Comment:
urgh, whose IDE isn't removing trailing whitespace??? cc @pyvista/developers

Reply:
Blame looks like dcbr and I.  I'll get better about checking the PRs.